the-collab-lab/tcl-20-whats-near-me
# App Details - custom icon & app name

## Description

- Created custom icon - a compass icon and named our app "Explore Your World" 
- Users should be able to install the app on their device homsecreens with the custom icon and app name
- Replaced generic react-app-icons with our custom compass favicon, logo.png files & compass.svg
- Added theme color, background color, title and description to app by updating the manifest.json and index.html files 

## Related Issue

Story: Create App Structure 

## Acceptance Criteria

- [x] Once this story is complete, users should be able to install the app on their device homescreens, identifiable as our app, not the generic create-react-app app

- [x] The tags/data points that should be updated include the following:

  - `public/index.html`
  - `<meta name="theme-color">`
  - `<meta name="description">`
  - `<title>`
  - `public/manifest.json`
  - `"short_name"`
  - `"name"`
  -` "icons"`
  - `"theme_color"`
  - `"background_color"`


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before
![image](https://user-images.githubusercontent.com/59700665/104780731-3b11fe80-574f-11eb-8d02-0b46e02202e1.png)

### After

![image](https://user-images.githubusercontent.com/59700665/104780835-67c61600-574f-11eb-92fa-4cd1b6165db8.png)

## Testing Steps / QA Criteria
Locally, you will be able to see the icon and name of the app on the Tab in your browser. Once the branch is approved and deployed, users should be able to install the app on their device homsecreens with the custom icon and app name.
